var/spack/repos/builtin/packages/r-fasterize/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RFasterize(RPackage):
+    """Provides drop-in replacement for rasterize() from the package raster."""
+
+    homepage = "https://cran.r-project.org/package=fasterize"
```

Comment:
This should reflect the URL, if there is one, in the DESCRIPTION file. This should also be reflected on the CRAN packages page. The URL, and homepage for spack, should be:

https://github.com/ecohealthalliance/fasterize